src/Api/WikibaseFactory.php

```diff
@@ -116,7 +116,7 @@ public function newValueFormatter() {
 	 * @return ItemMerger
 	 */
 	public function newItemMerger() {
-		return new ItemMerger( $this->api );
+		return new ItemMerger( new WikibaseApi( $this->api ) );
```

Comment:
Duplication... if you now change the `WikibaseApi` constructor params, you'll need change stuff here in a dozen places